package.json

```diff
@@ -4,66 +4,66 @@
   "private": true,
   "dependencies": {
     "@date-io/date-fns": "^1.3.11",
-    "@date-io/moment": "^2.0.1",
-    "@material-ui/core": "^4.8.3",
-    "@material-ui/icons": "^4.5.1",
-    "@material-ui/lab": "^4.0.0-alpha.42",
+    "@date-io/moment": "^2.10.8",
+    "@material-ui/core": "^4.11.3",
+    "@material-ui/icons": "^4.11.2",
+    "@material-ui/lab": "^4.0.0-alpha.57",
     "@material-ui/pickers": "^3.2.8",
-    "caniuse-lite": "^1.0.30001021",
-    "chart.js": "^2.9.3",
+    "caniuse-lite": "^1.0.30001192",
+    "chart.js": "^2.9.4",
     "classnames": ">=2.2.5",
-    "cropperjs": "^1.5.6",
-    "date-fns": "^2.6.0",
-    "downshift": "^4.0.6",
+    "cropperjs": "^1.5.11",
+    "date-fns": "^2.17.0",
+    "downshift": "^4.1.0",
     "history": "^4.10.1",
     "http-proxy-middleware": "^0.20.0",
-    "interweave": "^12.1.1",
-    "jwt-decode": ">=2.2.0",
-    "lodash": ">=4.17.11",
-    "luxon": "^1.21.3",
-    "moment": ">=2.24.0",
+    "interweave": "^12.7.0",
+    "jwt-decode": "^3.1.2",
+    "lodash": "^4.17.21",
+    "luxon": "^1.26.0",
+    "moment": "^2.29.1",
     "prop-types": ">=15.6.0",
-    "react": "^16.12.0",
-    "react-big-calendar": ">=0.21.0",
-    "react-chartjs-2": ">=2.7.4",
-    "react-cropper": ">=1.0.1",
+    "react": "^16.14.0",
+    "react-big-calendar": "^0.32.0",
+    "react-chartjs-2": "^2.11.1",
+    "react-cropper": "^2.1.7",
     "react-document-title": ">=2.0.3",
-    "react-dom": ">=16.2.0",
-    "react-dropzone": ">=10.1.5",
-    "react-ga": ">=2.3.5",
-    "react-graceful-image": ">=1.2.9",
-    "react-icons": ">=2.2.7",
-    "react-image-gallery": "^1.0.5",
+    "react-dom": "^17.0.1",
+    "react-dropzone": "^11.3.1",
+    "react-ga": "^3.3.0",
+    "react-graceful-image": "^1.5.0",
+    "react-icons": "^4.2.0",
```

Comment:
Four ones here also, not sure what react-dom and react-ga are used for, but the others look like they could affect ID Uploader and our icons

Reply:
`react-dom` is part of the core React functionality. React v17.x (and by extension `react-dom`) contains virtually no breaking changes, it was a purely internal update from them. You can read more about it in [the React team's official release announcement](https://reactjs.org/blog/2020/10/20/react-v17.html).

Reply:
`react-ga` is used for Google Analytics. The major update was simply to indicate that they are following semver strictly now, and contained no breaking changes.

Reply:
`react-icons` updated by two major versions. [v2.x->v3.x](https://github.com/react-icons/react-icons#migrating-from-version-2---3) had one breaking change which was changing the import path. We already use the new import path. 

[v3.x->v4.x](https://github.com/react-icons/react-icons/releases/tag/v4.0.0) had no breaking changes, but rather introduced a new package for projects configured differently from ours, which will not affect us. 

Therefore the `react-icons` update is clean.